CHANGELOG.md

```diff
@@ -5,9 +5,11 @@ All notable changes to this project will be documented in this file.
 The format of this file is based on
 [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [2.25.0] - TBD
+## unreleased
 
-TBD
+## [2.26.0] - 2024-03-01
```

Comment:
I think you'll be released 2.25 next, since that's what you published the beta for. The date should be set to Monday, since that's when you publish the actual release.